libsupport/include/katana/IndexID.h

```diff
@@ -7,7 +7,7 @@ namespace katana {
 
 /// A numeric ID type suitable for indexing. IndexIDs support:
 ///
-/// - Addition and subtraction of other IDs (the result of which is an ID)
+/// - Arithmetic operations with other IDs (the result of which is an ID)
```

Comment:
If you include them, you should say "bitwise operations". They are not arithmetic.

Reply:
Sure thing; I only did this because [this c++ doc site](https://en.cppreference.com/w/cpp/language/operator_arithmetic) groups them together as "arithmetic". Happy to supercede that with whatever folks like though.

Reply:
Include both. Just for people like me who don't think about things the same as the C++ docs. It will not confuse anyone I don't think.